migrations/src/9_deploy_razor_and_faucet.js

```diff
@@ -0,0 +1,33 @@
+const { NETWORK, SCHELLING_COIN_ADDRESS } = process.env;
+const {
+  deployContract,
+  appendDeploymentFile,
+  readOldDeploymentFile,
+} = require('../migrationHelpers');
+
+const { BigNumber } = ethers;
+const mintableSupply = (BigNumber.from(10).pow(BigNumber.from(27)));
+const deployRAZORAndFaucet = async () => {
+  if (NETWORK !== 'mainnet' && SCHELLING_COIN_ADDRESS === '') {
+    const RAZOR = await deployContract('RAZOR', [], [mintableSupply]);
```

Comment:
This is wrong
We have to pass initial supply here

Reply:
Ok, I will keep variable name as initialSupply because